CatsAndDogs/ContentLoader.swift

```diff
@@ -16,11 +16,18 @@ private extension Data {
 }
 
 class OnlineContentLoader {
-    let url = URL(string: "https://catsanddogs-kotlin-bff.herokuapp.com/schedule.json?from=2017-05-08T12:00:00-02:00")
+    let urlString = "https://catsanddogs-kotlin-bff.herokuapp.com/kotlinconf/schedule.json"
```

Comment:
I was not brave enough to add the date filter to the iOS app, a bit scared by storyboard :-/